pull_requests.md

```diff
@@ -77,41 +131,11 @@ In order to avoid situations like this, always remember that we review the code,
 
 * Submit PR
 * Tag relevant parties (using GitHub’s `@username`)
+  * Try to avoid tagging too many people in pull requests (in most cases 2 is the max).
+  * If your pull request contains changes to both frontend and backend code, tag someone from each team and tell them what to review.
 * Describe changes
 * Add comments about parts you need help with/are unsure about/etc.
+* If reviewers are neglecting you, you can bump the PR or send them a message.
```

Comment:
This is a great point, and I think we can take it further by pointing out that it's _your_ responsibility to get your PR merged, and you should regularly (gently) remind people that you need review.